ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/privacy/PrivateTransactionValidator.java

```diff
@@ -82,11 +81,13 @@ public PrivateTransactionValidator(final Optional<BigInteger> chainId) {
               transaction.getChainId().get(), chainId.get()));
     }
 
-    if (!chainId.isPresent() && transaction.getChainId().isPresent()) {
-      return ValidationResult.invalid(
-          REPLAY_PROTECTED_SIGNATURES_NOT_SUPPORTED,
-          "replay protected signatures is not supported");
-    }
+    // FIXME figure out why this is causing issues in
```

Comment:
This test is using a chain which has byzantium(?) starting at block 4(ish) - thus Pantheon is not expecting any CHAIN_ID in the transactions, and will reject them up until the 4th block has been mined - i.e. 20 seconds in.
If you use your new genesis file (with all 0's) - then you will need to use ChainID from the very beginning.